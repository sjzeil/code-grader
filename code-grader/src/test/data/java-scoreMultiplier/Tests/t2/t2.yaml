onSuccess: ["t2_OK", "t2_passed"]
onFail: ["t2_failed"]
failIf: "t10000_failed"