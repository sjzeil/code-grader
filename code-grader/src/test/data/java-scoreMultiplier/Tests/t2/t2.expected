The square root of 2.00 is 1.4142.
